todo_app/templates/index.html

```diff
@@ -11,10 +11,11 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
         {% for item in items %}
           <li>
-            {{ item.id }} - {{ item.title }} - {{ item.status }}
+            <form action="/completecard?id={{item.id}}&status={{ item.status }}" method="post" class="form-example">
+              <input type="submit" class="btn btn-success" value="Complete"> {{ item.id }} - {{ item.title }} - {{ item.status }} 
+            </form>
```

Comment:
The other pattern you'll often see here is, rather than specifying query parameters directly, adding hidden form inputs with defaults:
```suggestion
            <form action="/completecard" method="post" class="form-example">
              <input type="text" value="{{ item.id }}" name="id" hidden />
              <input type="text" value="{{ item.status }}" name="status" hidden />
              <input type="submit" class="btn btn-success" value="Complete"> {{ item.id }} - {{ item.title }} - {{ item.status }} 
            </form>
```

That will potentially be more useful if we're trying to combine the two recipient endpoints for the completing-cards action